examples/dt/uni04delta/edpm-pre-ceph/nodeset/values.yaml

```diff
@@ -0,0 +1,160 @@
+        #       podman login -u <registry_username> -p <registry_password> registry.redhat.io
+        edpm_logrotate_crond_image: '{{ registry_url }}/openstack-cron:{{ image_tag }}'
+        edpm_network_config_hide_sensitive_logs: false
+        edpm_network_config_template: |
+          ---
+          {% set mtu_list = [ctlplane_mtu] %}
+          {% for network in nodeset_networks %}
+          {{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
+          {%- endfor %}
+          {% set min_viable_mtu = mtu_list | max %}
+          network_config:
+          - type: ovs_bridge
+            name: {{ neutron_physical_bridge_name }}
+            mtu: {{ min_viable_mtu }}
+            use_dhcp: false
+            dns_servers: {{ ctlplane_dns_nameservers }}
+            domain: {{ dns_search_domains }}
+            addresses:
+            - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
+            routes: {{ ctlplane_host_routes }}
+            members:
+            - type: interface
+              name: nic2
+              mtu: {{ min_viable_mtu }}
+              # force the MAC address of the bridge to this interface
+              primary: true
+          {% for network in nodeset_networks %}
+            - type: vlan
+              mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+              vlan_id: {{ lookup('vars', networks_lower[network] ~ '_vlan_id') }}
+              addresses:
+              - ip_netmask:
+                  {{ lookup('vars', networks_lower[network] ~ '_ip') }}/{{ lookup('vars', networks_lower[network] ~ '_cidr') }}
+              routes: {{ lookup('vars', networks_lower[network] ~ '_host_routes') }}
+          {% endfor %}
+        edpm_iscsid_image: '{{ registry_url }}/openstack-iscsid:{{ image_tag }}'
+        edpm_neutron_metadata_agent_image: '{{ registry_url }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}'
+        edpm_nodes_validation_validate_controllers_icmp: false
+        edpm_nodes_validation_validate_gateway_icmp: false
+        edpm_nova_compute_container_image: '{{ registry_url }}/openstack-nova-compute:{{ image_tag }}'
+        edpm_nova_libvirt_container_image: '{{ registry_url }}/openstack-nova-libvirt:{{ image_tag }}'
+        edpm_ovn_controller_agent_image: '{{ registry_url }}/openstack-ovn-controller:{{ image_tag }}'
+        edpm_selinux_mode: enforcing
+        edpm_sshd_allowed_ranges:
+          - 192.168.122.0/24
+        edpm_sshd_configure_firewall: true
+        enable_debug: false
+        gather_facts: false
+        image_tag: current-podified
+        neutron_physical_bridge_name: br-ex
+        neutron_public_interface_name: eth0
+        registry_url: quay.io/podified-antelope-centos9
+        edpm_ceph_hci_pre_enabled_services:
+          - ceph_mon
+          - ceph_mgr
+          - ceph_osd
+          - ceph_rgw
+          - ceph_nfs
+          - ceph_rgw_frontend
+          - ceph_nfs_frontend
+        storage_mtu: 9000
```

Comment:
VA1 does the same because it overrides the default MTU of 1500 for the storage network and adds the storage management network. Both uniDelta and va1 deploy EDPM nodes which host Ceph.

The storage management network does not need to be created in IPAM on the controller node. It's only used by nodes hosting ceph for ceph to sync it's internal data. I don't think we need to block this over storage_mgmt_ network being here. If you have an more dataplane-operator-netconfig-like way to do this, then let me know and we can do a follow up patch on va1 and uniDelta.

https://github.com/openstack-k8s-operators/architecture/blob/main/examples/va/hci/edpm-pre-ceph/nodeset/values.yaml#L95-L99

Reply:
yes was not blocking, just raised this so be checked later